schedule/sles4sap/migration/sles4sap_horizontal_migration.yaml

```diff
@@ -12,5 +12,5 @@ schedule:
   - console/system_prepare
   - sles4sap/migrate_sles_to_sles4sap
   - sles4sap/patterns
-  - sles4sap/sapconf
   - sles4sap/saptune
+  - sles4sap/sapconf
```

Comment:
I don't understand why this change is necessary. I take it the test is failing, but why?

Horizontal migration means from plain SLES to SLES for SAP Applications, and SLES does not have saptune ... so why is this an issue here?

Are we certain that this change won't be ignoring an issue?

Reply:
The same problem we had with other migrations: Angela's sapconf testsuite does a lot of things like disabling tuned which comes enabled by default by saptune installation.